dashboard/config/database.yml

```diff
@@ -1,8 +1,4 @@
 <%
-deployment_path = File.expand_path('../../deployment.rb', __FILE__)
-deployment_path = File.expand_path('../../../../deployment.rb', __FILE__) unless File.file?(deployment_path)
-require deployment_path
```

Comment:
I traced through the `require` graph currently used by our project, and it looks like [`deployment.rb`](https://github.com/code-dot-org/code-dot-org/blob/staging/deployment.rb) should always already be required by our Rails app in [`application.rb`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/application.rb#L2), which is included on every Rails bootup.

Requiring relative file from within `database.yml` was causing issues because when this string is evaluated as a string through the ERB filter, `__FILE__` resolves to the current working directory rather than the location of the file. Removing this unnecessary require altogether seemed like the best fix.